src/Exceptions/ChartMogulException.php

```diff
@@ -38,6 +39,9 @@ public function __construct($message, ResponseInterface $response = null, \Excep
             $json = json_decode($body, true);
 
             $this->response = json_last_error() === JSON_ERROR_NONE? $json : $body;
+            
+            // Adding body to message to get the whole error message to API user.
+            $message = $message . " " . $body;
```

Comment:
Do think we should do something like Response:\n ? Make it clearer what's the response body and what's the exception message?